pkg/controller/sql/controller.go

```diff
@@ -160,14 +169,22 @@ func (r *SqlReconciler) requeueResult(ctx context.Context, resource Resource, er
 		return ctrl.Result{RequeueAfter: resource.RequeueInterval().Duration}, nil
 	}
 	if r.RequeueInterval > 0 {
+		requeueInterval := r.requeueIntervalWithOffset()
 		if r.LogSql {
 			log.FromContext(ctx).V(1).Info("Requeuing SQL resource")
 		}
-		return ctrl.Result{RequeueAfter: r.RequeueInterval}, nil
+		return ctrl.Result{RequeueAfter: requeueInterval}, nil
```

Comment:
For consistency, I think we should probably add the offset here as well:
- https://github.com/mariadb-operator/mariadb-operator/blob/b63954b40fd38d7913c46417dfb0a0b67b8b9e4f/pkg/controller/sql/controller.go#L160

This applies to `requeueInterval` provided in the CR, whereas the one here is the default set by the controller. One could always leave the offset to 0 (default) without affecting the current behaviour.